addons/l10n_it_edi/models/ir_mail_server.py

```diff
@@ -136,11 +136,11 @@ def _attachment_invoice(self, msg_txt):
                     self._message_receipt_invoice(split_underscore[1], attachment)
                 else:
                     att_filename = attachment.fname
-                    match = re.search("([A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.(xml.p7m|xml))", att_filename)
+                    match = re.search("[A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.((?i:xml.p7m|xml))", att_filename)
```

Comment:
Something that bothers me is that you're redefining the same regex here. 
It's too easy to update one and forget the other which would lead to inconsistencies and issues.
Could you do as Paolo said please? `match = self.env['account.edi.format']._check_filename_is_fattura_pa(att_filename)`

Reply:
It's ok with me if you prefer to avoid changing method calls in stable, it's safer anyway.
Just remember to change this in 16.0/master for the future then :smile:

Reply:
@vin-odoo ok, then we keep it and we change further on the line

Reply:
Actually we do not need to, because the mail system is not there in v16.